AmplicaLabs/social-app-template
# feat: Update IPFS gateway URL in env template

# Purpose
The goal of this PR is get SAT working with zero changes.

## Solution

Some environment variables were misconfigured in the templates.
This corrects the variables so that graph-service works correctly and ipfs content is resolved correctly.

### Change summary
- `env.content-publishing-service.template` update `IPFS_GATEWAY_URL`
- `env.graph-service.template` update `GRAPH_ENVIRONMENT_TYPE`


## Steps to Verify
1. Follow steps in README and make a post.
2. Wait 30-60 seconds (allow content-watcher interval to catch the new content on chain).
3. Verify that ipfs links are resolved and jpg can be displayed under `My Posts`
4. Logout and create a new user and login with that new user.
5. Find the original user under the `Discover` tab click to their profile.
6. Follow the original user and verify that the follow succeeds (the spinner should eventually stop and the follow button will switch to unfollow)
